packages/lit-dev-content/site/tutorials/content/working-with-lists/00.md

```diff
@@ -0,0 +1,24 @@
+This tutorial will be exploring different ways of rendering a list of items in
```

Comment:
Do we want to say "different ways of rendering a list" or "different ways of rendering lists"?

You can render more than one list at once, so I'd lean towards plural. @arthurevans ?